src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        String randomColor = ColorSupplier.getRandomColor();
+        // if getRandomColor() not be static, will be next problem:
+        // Non-static method 'getRandomColor()' cannot be referenced from a static context
+        final String[] figures = {"square",
+                                  "rectangle",
+                                  "right triangle",
+                                  "circle",
+                                  "isosceles trapezoid"};
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level